packages/graphql-serve/src/commands/printSchema.ts

```diff
@@ -1,5 +1,5 @@
 import yargs from 'yargs';
-import * as chalk from 'chalk';
```

Comment:
@machi1990 I know that you changed this back, but I was getting a build error:

`Unsafe call of an any typed value`. Any reason the import alias is preferred here?

Reply:
This was changed in https://github.com/aerogear/graphback/pull/2026 since the `printSchema` command was not working.

Reply:
I've accepted the change without re-verifying this part as it seems unrelated. I trust you on that and the command should work.

Reply:
Okay it is not working for me now - what I will do is disable that line with eslint